docs/going-into-production.rst

```diff
@@ -0,0 +1,357 @@
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+You can configure a node that handles cluster management (i.e., is
+master-eligible) but does not handle query execution loads like this:
+
+.. code-block:: yaml
+
+    node.master: true
+    node.data: false
+
+A good name for this node might be:
+
+.. code-block:: yaml
+
+    node.name: node-03-m
+
+Here, ``node`` is a base name with a sequence number of ``03``. Every node in
+the cluster should have a unique sequence number, independent of the node type.
+The letter ``m`` indicates that this node has ``node.master`` set to ``true``.
+
+
+.. _prod-node:
+
+Request handling nodes
+~~~~~~~~~~~~~~~~~~~~~~
+
+You can configure a node that handles client requests but does not handle query
+execution loads or cluster management (i.e., is not master-eligible) like this:
+
+.. code-block:: yaml
+
+    node.master: false
+    node.data: false
+
+A good name for this node might be:
+
+.. code-block:: yaml
+
+    node.name: node-04
+
+Here, ``node`` is a base name with a sequence number of ``04``. Every node in
+the cluster should have a unique sequence number, independent of the node type.
+The absense of any additional letters indicate that ``node.master`` and
+``node.data`` are set to ``false``.
+
+
+.. _prod-config-hostname:
+
+Bind nodes to logical hostnames
+===============================
+
+By default, CrateDB binds to the loopback address (i.e., `localhost`_). It
+listens on port 4200-4299 for HTTP traffic and port 4300-4399 for node-to-node
+communication. Because CrateDB uses a port range, if one port is busy, it will
+automatically try the next port.
+
+When using :ref:`multiple hosts <multi_node_setup>`, nodes must bind to a
+non-loopback address.
+
+.. CAUTION::
+
+      Never expose an unprotected CrateDB node to the public internet
+
+You can bind to a non-loopback address with the `network.host`_ setting in your
+`configuration`_ file, like so:
+
+.. code-block:: yaml
+
+    network.host: node-01-md.acme-prod.internal.example.com
+
+You can break down that hostname, as follows:
+
+- ``example.com`` -- Your example domain name
+- ``internal`` -- Your internal private network
+- ``acme-prod`` -- The cluster name
+- ``node-01-md`` -- The node name
+
+When CrateDB is bound to a non-loopback address, CrateDB will enforce the
+:ref:`bootstrap checks <bootstrap-checks>`. These checks may require changes to
+your operating system configuration.
+
+.. SEEALSO::
+
+    `Host settings`_
+
+
+.. _prod-config-paths:
+
+Configure persistent data paths
+===============================
+
+By default, CrateDB keeps data under the `CRATE_HOME`_ directory (which
+defaults to the installation directory). When you upgrade CrateDB, you will
+have to switch to a new installation directory.
+
+Instead of migrating data by hand each time, you should move the data
+directories off to a persistent location. You can do this using the
+`CRATE_HOME`_ environment variable and the `path settings`_ in your
+`configuration`_ file.
```

Comment:
Should we maybe point out that the distribution packages include a service file that typically set up `CRATE_HOME` correctly, and that then it is only necessary to set the `path_settings`?

Reply:
done